Sources/RingPublishingTracking/Public/RingPublishingTrackingConfiguration.swift

```diff
@@ -49,12 +53,14 @@ public struct RingPublishingTrackingConfiguration {
                 apiUrl: URL? = nil,
                 applicationRootPath: String,
                 applicationDefaultStructurePath: [String]? = nil,
-                applicationDefaultAdvertisementArea: String? = nil) {
+                applicationDefaultAdvertisementArea: String? = nil,
+                applicationAdvertisementSite: String) {
```

Comment:
zróbmy opcjonalne bo robimy wersje bugfix, tak to będzie niekompatybilne wstecz
czyli
applicationAdvertisementSite: String? = nil